src/components/splash_screen.module.css

```diff
@@ -0,0 +1,70 @@
+.root {
+  position: absolute;
+
+  top: 0;
+  left: 0;
+
+  width: 100vw;
+  height: 100vh;
+
+  background-color: rgb(4, 92, 252);
+  overflow: hidden;
+
+  z-index: 10;
+}
+
+.splash {
+  position: absolute;
+
+  top: 50%;
+  left: 50%;
+
+  background-color: rgb(4, 92, 252);
+  transform: translate(-50%, -50%);
+
+  border-radius: 50%;
+
+  z-index: 10;
+}
+
+.logo {
+  position: absolute;
+
+  text-align: center;
+
+  top: 50%;
+  left: 50%;
+
+  transform: translateX(-50%) translateY(-50%);
+
+  z-index: 11;
+}
+
+.logo svg {
+  width: calc(100% - 80px);
+  max-width: 495px;
+}
+
+@media only screen and (orientation: portrait) {
+  .splash {
+    width: 150vh;
+    height: 150vh;
+  }
+}
+
+@media only screen and (orientation: landscape) {
+  .splash {
+    width: 150vw;
+    height: 150vw;
+  }
+}
```

Comment:
What's the idea here? Wouldn't `width: 100vw` and `height: 100vh` work for every case?

Reply:
This was Justo's code. Let me double check and I'll get back to you.

Reply:
Yeah, this has some logic to it. Basically this sets the splash screen to be much larger than the actual screen, otherwise, when morphing it would look pretty strange. The difference between the `vw` and `vh` is to basically use the largest unit, on portrait `height > width` so we use height. Same stuff for landscape.

Reply:
This way, when we start the morph, the planet would be bigger than screen giving us a smoother effect.